dotnet/targets/Xamarin.Shared.Sdk.props

```diff
@@ -22,5 +22,11 @@
 		<_XamarinSdkRoot Condition="'$(_XamarinSdkRoot)' == ''">$(_XamarinSdkRootDirectory)</_XamarinSdkRoot>
 		<!-- _XamarinSdkRootOnMac this should be passed to tasks that need to access the Xamarin Sdk dir on the Mac, this value will be overriden from Windows -->
 		<_XamarinSdkRootOnMac>$(_XamarinSdkRoot)</_XamarinSdkRootOnMac>
+
+		<!-- We don't need any runtime configuration files -->
+		<GenerateRuntimeConfigurationFiles Condition="'$(GenerateRuntimeConfigurationFiles)' ==''">false</GenerateRuntimeConfigurationFiles>
```

Comment:
You will actually need it at some point. There's a Mono task that converts the JSON into binary blob and that can be passed to the Mono runtime. I recently implemented it in AppleAppBuilder on the dotnet/runtime side.

Reply:
PR (for example usage): https://github.com/dotnet/runtime/pull/53172
Tracking issue: https://github.com/dotnet/runtime/issues/49237

It's necessary for features like `EnableUnsafeUTF7Encoding` and `EnableUnsafeBinaryFormatterSerialization`.

Reply:
So I guess we'll have to:

* Keep generating *.runtimeconfig.json file
* Use Mono's task to convert the json to a binary blob.
* Bundle the blob in the .app
* Don't bundle the json in the app.

Is that correct?

Reply:
I've filed #11745 to track this.

Reply:
Sounds correct to me.